editor/assets/stylesheets/variables.scss

```diff
@@ -35,3 +35,4 @@ $editor-font: "Noto Serif", serif;
 $editor-html-font: Menlo, Consolas, monaco, monospace;
 $editor-font-size: 16px;
 $editor-line-height: 1.8em;
+$item-spacing: 10px;
```

Comment:
`item` here seems very generic - where exactly would we expect this spacing to apply? Also to controls in the block-level toolbar?